README.md

```diff
@@ -176,8 +146,7 @@ Generally, keys specified in `datapackage.json` take precedence over data loaded
 For pages, the following precedence should be used:
 
 1) if `datapackage.json` `pages` key exists, pages loaded from there.
```

Comment:
We no longer have the pages key, but, we do have the optional mainPageURL and mainPageTs, so can mention those above as part of datapackage.jsonl